roles/confluent.kafka_broker/tasks/rbac_rolebindings.yml

```diff
@@ -0,0 +1,26 @@
+---
+- name: RBAC Setup Tasks
+  include_tasks: ../../tasks/rbac_setup.yml
+  vars:
+    user: "{{kafka_broker_user}}"
+    group: "{{kafka_broker_group}}"
+
+- name: Grant role System Admin to Additional Kafka Broker System Admins
```

Comment:
As this request (to grant some role to a certain user) is done in many places around the code, would it be possible to extract it into a function or something (not sure what ansible's concept of code reuse looks like)?

Reply:
I think what you are asking for is a generic create role binding playbook, which is a good idea. I think we even have something like that in our 2021 planning. That's not the goal of this PR though